### DISCUSSION TITLE
How to use brand colors with inline CSS?

### DISCUSSION THREAD

**Discussion by MincePie** [chakra-ui/chakra-ui#6029] [Q&A] [answered]
I'm trying to figure out how to use brand colors with inline CSS styling.

When I try:

```
<Heading
                as="h1"
                size="3xl"
                fontWeight="extrabold"
                px="3rem"
                lineHeight="1.2"
                letterSpacing="tight"
                color="brand.magenta"
                paddingInline="0"
              >
                Two <span style={{color: "#002a32"}}>Words</span> 
              </Heading>
```

I can override the colour of the Heading with the span style for the second word.

I want to be able to use a brand color, so the span would be: <span style={{color: "brand.darkGreen"}}>

Is that possible?

**Comment by hugobqd** [comment, marked as answer]
if your brand colors are stored in the theme object, your could use it like this : 
`<Box as='span' color="brand.darkGreen">Words</Box>`

**Reply by MincePie** [reply]
Thanks @hugobqd
